src/components/BlockingViews/BlockingView.tsx

```diff
@@ -31,20 +28,46 @@ type BlockingViewProps = {
     /** Whether we should show a link to navigate elsewhere */
     shouldShowLink?: boolean;
 
+    /** Function to call when pressing the navigation link */
+    onLinkPress?: () => void;
+
+    /** Whether we should embed the link with subtitle */
+    shouldEmbedLinkWithSubtitle?: boolean;
+
+    /** Render custom subtitle */
+    CustomSubtitle?: React.ReactElement;
```

Comment:
@tienifr Using camelCase convention

Reply:
`PascalCase` is correct for component props